tsdb/raw.go

```diff
@@ -93,6 +93,7 @@ func (e *RawExecutor) execute(out chan *models.Row) {
 	// Keep looping until all mappers drained.
 	var err error
 	for {
+		timeout := time.After(30 * time.Second)
```

Comment:
yeah, I wasn't sure really where to put the timeout in this scenario. Maybe we don't need one for raw mappers?  Or drop it down one more `for`?  Thoughts?